pom.xml

```diff
@@ -63,6 +63,9 @@ under the License.
 
         <flink.parent.artifactId>flink-connector-jdbc-parent</flink.parent.artifactId>
         <flink.forkCountITCase>1</flink.forkCountITCase>
+        <!-- These 2 properties should be removed together with upgrade of flink-connector-parent to 1.1.x -->
+        <flink.surefire.baseArgLine>-XX:+UseG1GC -Xms256m -XX:+IgnoreUnrecognizedVMOptions ${surefire.module.config}</flink.surefire.baseArgLine>
+        <surefire.module.config/>
```

Comment:
Why is is this a part of the PR?
It is already in main

I guess you need to rebase

Reply:
yes there was a merge conflict I will sort this out